packages/backend/src/webhook/service.ts

```diff
@@ -295,7 +302,22 @@ export function finalizeWebhookRecipients(
     tenantIdSet.add(config.operatorTenantId)
   }
 
-  return [...tenantIdSet.values()].map((tenantId) => ({
+  let recipients: Pick<Webhook, 'recipientTenantId' | 'metadata'>[] = [
+    ...tenantIdSet.values()
+  ].map((tenantId) => ({
     recipientTenantId: tenantId
   }))
+
+  if (metadata?.isCardPayment && config.posServiceUrl) {
```

Comment:
we should do a warn log at if its a card payment but the posServiceUrl is not set